### PR TITLE
documentation(EJ2-71059):New documentation for autoFit in DataGrid

### PR DESCRIPTION
**Pull request by MUTHUSELVISF3942**
New documentation for autoFit in DataGrid

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/columns/auto-fit-columns.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-control: Auto Fit Columns
+control: AutoFit Columns
 publishingplatform: ##Platform_Name##
 documentation: ug
 ---
 
-# AutoFit Specific Columns
+# AutoFit Columns
 
-The [`autoFitColumns`](https://ej2.syncfusion.com/documentation/api/grid/#autofitcolumns) method resizes the column to fit the widest cell's content without wrapping. You can autofit a specific column at initial rendering by invoking the [`autoFitColumns`](https://ej2.syncfusion.com/documentation/api/grid/#autofitcolumns) method in [`DataBound`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_DataBound) event.
```
Comment (jmfarook7) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/columns/auto-fit-columns.md:13]: api casing is incorrect. Need to mention the API name as Pascal came in MVC and core

**Review by jmfarook7 — CHANGES_REQUESTED** [review]
all the API casing is incorrect. Need to mention the API name as Pascal came in MVC and core

**Comment by MUTHUSELVISF3942** [comment]
rebuild

**Comment by MUTHUSELVISF3942** [comment]
rebuild

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/columns/auto-fit-columns.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+The Grid has a feature that allows it to automatically adjust column widths based on the maximum content width of each column when you double-click on the resizer symbol located in a specific column header.  This feature ensures that all data in the grid rows is displayed without wrapping. To use this feature, you need to inject the **Resize** module and enable the resizer symbol in the column header by setting the [AllowResizing](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_AllowResizing) property to true in the grid.
+
+## Resizing a Column to fit its content using autoFit method
+
+The [AutoFitColumns](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.Grid.html#Syncfusion_EJ2_Grids_Grid_AutoFitColumns) method resizes the column to fit the widest cell's content without wrapping. You can autofit a specific column at initial rendering by invoking the `AutoFitColumns` method in [DataBound](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridBuilder-1.html#Syncfusion_EJ2_Grids_GridBuilder_1_DataBound_System_String_) event.
```
Comment (jmfarook7) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/columns/auto-fit-columns.md:17]: method name is not start with pascal case

**Review by jmfarook7 — COMMENTED** [review]
method name as incorrect case

**Review by jmfarook7 — APPROVED** [review]
fine

**Pull request merged**